src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -0,0 +1,87 @@
+import Plugin from 'src/plugin-system/plugin.class';
+
+export default class CheckoutPlugin extends Plugin {
+
+    init() {
+        /* global adyenCheckoutConfiguration, AdyenCheckout */
+        /* eslint-disable no-unused-vars */
+
+        const paymentMethodTypeHandlers = {
+            'scheme': 'handler_adyen_cardspaymentmethodhandler'
+        };
+
+        var ADYEN_CHECKOUT_CONFIG = {
+            locale: adyenCheckoutConfiguration.locale,
+            originKey: adyenCheckoutConfiguration.originKey,
+            environment: adyenCheckoutConfiguration.environment,
+            showPayButton: false,
+            paymentMethodsResponse: JSON.parse(adyenCheckoutConfiguration.paymentMethodsResponse)
+        };
+
+        window.adyenCheckout = new AdyenCheckout(ADYEN_CHECKOUT_CONFIG);
+
+        var placeOrderAllowed = false;
+        var data;
+
+        // use this object to iterate through the stored payment methods
+        var paymentMethods = window.adyenCheckout.paymentMethodsResponse.paymentMethods;
+
+        // Iterate through the payment methods list we got from the adyen checkout component
+        paymentMethods.forEach(function (paymentMethod) {
+            //  if the container doesn't exits don't try to render the component
+            var paymentMethodContainer = $('[data-payment-method="' + paymentMethodTypeHandlers[paymentMethod.type] + '"]');
+
+            // container doesn't exist, something went wrong on the template side
```

Comment:
This is a great code comment. Not just telling what, but also why.

Reply:
Maybe because it was copy-pasted from @cyattilakiss in PS 😂